Add manifest for building Krogoth

leon-anavi opened the pull request:
Add manifest file for building Poky release Krogoth with meta-updater

leon-anavi committed:
krogoth.xml: Repo manifest for YP release Krogoth

Signed-off-by: Leon Anavi <leon.anavi@konsulko.com>

leon-anavi committed:
krogoth.xml: Use Krogoth branch for meta-updater & rpi

Use branches krogoth of meta-updater-raspberrypi,
meta-updater and meta-raspberrypi for release
Krogoth of the Yocto Project.

Signed-off-by: Leon Anavi <leon.anavi@konsulko.com>

OYTIS commented on krogoth.xml:
```diff
@@ -0,0 +1,27 @@
+	<!-- Updater layers -->
+	<project path="meta-updater" name="advancedtelematic/meta-updater" remote="github" revision="krogoth" />
+	<project path="meta-updater-raspberrypi" name="advancedtelematic/meta-updater-raspberrypi" remote="github" revision="krogoth" />
+	<project path="meta-updater-minnowboard" name="advancedtelematic/meta-updater-minnowboard" remote="github" revision="morty" />
```
I would change revisions of all the meta-updater-* layers to krogoth for consistency. Just created dummy krogoth branches where they were missing.

pattivacek commented:
Works for me. If you make Anton's suggested changes, I'll approve.

leon-anavi committed:
krogoth.xml: Use branch Krogoth for all targets

Use branch Krogoth for all supported devices.

Suggested-by: Anton Gerasimov <anton@advancedtelematic.com>
Signed-off-by: Leon Anavi <leon.anavi@konsulko.com>

leon-anavi commented:
@OYTIS @patrickvacek thank you for the feedback. I have updated the xml file and I have added @OYTIS in suggested-by field in the latest git commit.

Thanks, Leon
